bridge/README.md

```diff
@@ -1,36 +1,27 @@
 # Keptn Bridge
 
-Keptn bridge allows to browse the Keptn events.
+The Keptn Bridge is the Keptn user interface. It provides a dashboard with information about all projects and services managed by Keptn.
 
-Note that yarn dependencies are separated into two parts. Root level `package.json` contains dependencies for angular and other general requirements. Express server dependencies are located inside `package.json` located in server folder.
+Note that yarn dependencies are separated into two parts. The root level `package.json` contains dependencies for Angular and other general requirements. The Express server dependencies are located inside the `server/package.json` file.
 
 ## Installation
 
-The Keptn bridge is installed as a part of [Keptn](https://keptn.sh).
-
-### Deploy in your Kubernetes cluster
-
-To deploy the current version of the bridge in your Keptn Kubernetes cluster, use the file `deploy/service.yaml` from this repository and apply it:
-
-```console
-kubectl apply -f deploy/service.yaml
-```
+The Keptn bridge is installed as a part of [Keptn](https://keptn.sh). To get started with Keptn, please follow the [documentation](https://keptn.sh/docs/install/). If you want to install the most recent development version, follow the ["Install master branch" guide](https://github.com/keptn/keptn/blob/master/docs/developer/install_master.md).
 
 ### Environment variables
 - `ENABLE_VERSION_CHECK` - If disabled, versions.json is not loaded and the version info will not be displayed.
 - `SHOW_API_TOKEN` - If disabled, the API token will not be shown in the Bridge info.
-- `PROJECTS_PAGE_SIZE` - Determines, how many projects will be fetched for the Bridge. If not set, it defaults to 50.
-- `KEPTN_INSTALLATION_TYPE` - Can take the values: `QUALITY_GATES`, `CONTINUOUS_OPERATIONS`, `CONTINUOUS_DELIVERY` and determines the mode in which the Bridge will be started. If only `QUALITY_GATES` is set, only functionalities and data specific for the Quality Gates Only use case will be displayed.
+- `PROJECTS_PAGE_SIZE` - Determines how many projects will be fetched for the Bridge. If not set, it defaults to 50.
+- `KEPTN_INSTALLATION_TYPE` - Can take the values: `QUALITY_GATES`, `CONTINUOUS_OPERATIONS`, and `CONTINUOUS_DELIVERY` and determines the mode in which the Bridge will be started. If only `QUALITY_GATES` is set, only functionalities and data specific for the Quality Gates Only use case will be displayed.
 
 ### Setting up Basic Authentication
 
-Keptn Bridge comes with a very simple basic authentication feature, which can be controlled by setting the following two environment variables:
+Keptn Bridge comes with a straightforward basic authentication feature, which can be controlled by setting the following two environment variables:
```

Comment:
Keptn Bridge can use basic authentication, the following two environment variables define the values: